model-optimizer/extensions/back/NormalizeToNormalizeL2.py

```diff
@@ -28,6 +28,8 @@ class NormalizeToNormalizeL2(BackReplacementPattern):
     enabled = True
     force_clean_up = True
 
+    id = 'NormalizeToNormalizeL2_transform'
```

Comment:
I suggest to implement completely different approach. Instead of adding `id` attribute for each and every class we want to enable/disable let's change the function `_update` from the "model-optimizer/mo/utils/class_registration.py" so it will be able to handle the `MO_DISABLED_TRANSFORMS` and `MO_ENABLED_TRANSFORMS` with fully defined class names. So you can add "extensions.back.NonmalizeToNormalizeL2. NormalizeToNormalizeL2" to one of the environment variables and this transformation will be disabled/enabled. 

To get the fully defined class name for the class `c` you can write the following code: `".".join([c.__module__, c.__name__])`

Reply:
Ok, but I have concern about this - if we change this approach we will be needed to find all usages of `MO_DISABLED_TRANSFORMS` and `MO_EABLED_TRANSFORMS` in our e2e tests and other places and update their values.

Reply:
As we agreed during the meeting the behaviour with `id` should be kept as is. We should "extend" the current behaviour to work with module names.

Reply:
Ok, done.